docs/watchtower.md

```
@@ -0,0 +1,260 @@
+Installing or upgrading the operator from inside the pod needs more again, and is
+not granted today.
+
+# Open Questions/Notes
+
+- How to test a deploy for this? Watchtower is only operator v2 compatible, do we have dedicated instances on operator v2?
+- How to deploy with Orca?
+- Need to pin replicas: 1 so that reconnects to land in a pod that has never seen a run and has no history
+- Probably want to remove some functionality from Watchtower for this deployment like context selecting and `teardown`
+- The Watchtower service is going to need greater permissions to install/upgrade the operator from inside the pods
+
+### Notes from Claude
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)